### PR TITLE
Add keycodes for pointing device scroll-mode hold

### PR DESCRIPTION
**Pull request by vnegnev**
PR adds keycodes that behave like the existing pointing device scroll-mode toggles, except they change the scroll-mode state both when pressed and when released, i.e. the state is changed while they are held down, and reverts when released. They do not interact with the EEPROM.

Also includes these keycodes, and the original scroll-mode toggles, in the set of keycodes that do not exit the automouse layer, so that they can be used with automouse.

Tested on a left-trackpoint Svalboard.

### CODE REVIEW

File: keyboards/svalboard/keymaps/keymap_support.c [hunk 1]
```diff
@@ -271,8 +277,14 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
                 mouse_mode(false);
         }
     }
-    if (record->event.pressed) {
+    if (record->event.pressed) { // key pressed
         switch (keycode) {
+            case SV_LEFT_SCROLL_HOLD:
```
Comment (ilc) [inline keyboards/svalboard/keymaps/keymap_support.c:261]: Don't modify global_saved_values without calling write_eeprom_kb().

Make another global variable and, | the two values together.

**Review by ilc — CHANGES_REQUESTED** [review]
Please make the needed changes.

File: keyboards/svalboard/keymaps/keymap_support.c (hunk 1 above)
Comment (vnegnev) [reply keyboards/svalboard/keymaps/keymap_support.c:261]: Done

**Comment by vnegnev** [comment]
I've rebased off the docs branch, modified the code as advised to avoid using the saved values, and added v0.0.10 as a tag. Tested on a left-trackpoint Svalboard.

File: keyboards/svalboard/keymaps/keymap_support.h [hunk 2]
```diff
@@ -28,6 +28,8 @@ enum my_keycodes {
     SV_RIGHT_DPI_DEC,
     SV_LEFT_SCROLL_TOGGLE,
     SV_RIGHT_SCROLL_TOGGLE,
+    SV_LEFT_SCROLL_HOLD,
```
Comment (ilc) [inline keyboards/svalboard/keymaps/keymap_support.h]: Key codes always go at the end, or they break all existing maps.

**Review by ilc — CHANGES_REQUESTED** [review]
Minor change, but needed,

File: keyboards/svalboard/keymaps/keymap_support.h (hunk 2 above)
Comment (vnegnev) [reply keyboards/svalboard/keymaps/keymap_support.h]: Done, good catch. I've updated the order in both keymap_support.h and vial.mk.

**Review by ilc — COMMENTED** [review]
LGTM.

**Review by ilc — APPROVED** [review]

**Pull request merged**